### ISSUE TITLE
Handle share URLs locally instead of sending to the server

### ISSUE THREAD

**Issue by rugk** [tyrasd/overpass-turbo#338]
You sometimes generate long URLs, which – as your UI indicates – can be problematic. However, why not use anchors `#`, which are not transmitted to the server, instead of query parameters? This would make it possible to work around server limitations on the URL length and I think browsers do not have a limit or a very large one.

**Comment by tyrasd** [comment]
+1, sounds like a good idea. just a side note: the server is not the only limiting factor, browsers have their own limits: https://stackoverflow.com/a/417184/1627467

**Comment by rugk** [comment]
Yes, but anchor tags can be much longer than query stuff (as the Stackoverflow answer also mentions…).
